ToDoWebApp/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ToDoWebApp.Data.Intefaces;
+using ToDoWebApp.Models;
+
+namespace ToDoWebApp.Controllers
+{
+    public class CategoryController : Controller
+    {
+
+        public static List<Category> asList;
+        public static bool pereita;
```

Comment:
Variable naming. Code should be consistent. Either every method, variable, class etc. should be named Lithuanian or English - not both in different places.